src/components/LyricViewer.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './LyricViewer.css';
+
+interface LyricViewProps {
+  songNumber: number;
+}
+
+/**
+ * Lyric Viewer React Functional Component.
+ */
+const LyricViewer: React.FC<LyricViewProps> = (props) => {
+
+  var data;
+  try {
+    data = require(`../../resources/Songs_&_Hymns_Of_Life/metadata/${props.songNumber}.json`);
+  } catch {
+    return <h1 className="center">No Song Found</h1>;
+  }
+  return (
+    <div>
+      <h2 className="center">{data['title']}</h2>
+      <b>
+          {/* I have no idea how to print these lyrics! 
+          
+          fyi - a couple of the songs use capitals verse numbers for some reason (ie: V1 instead of v1)
+          */}
+        {data['lyrics']['v1']}
+      </b>
+
+      <footer>Author: {data['author'] == '' ? 'Unknown' : data['author']}</footer>
+    </div>
```

Comment:
I couldn't figure out how to display all the verses properly here.

The lyrics are stored with each verse being a separate object and each line as part of an array within that verse. Tricky tricky.

Reply:
the `V1` vs `v1` thing will need to be corrected at some point.